rotkehlchen/db/eth2.py

```diff
@@ -185,11 +197,19 @@ def add_validators(self, validators: List[Eth2Validator]) -> None:
         cursor = self.db.conn.cursor()
         cursor.executemany(
             'INSERT OR IGNORE INTO '
-            'eth2_validators(validator_index, public_key) VALUES(?, ?)',
+            'eth2_validators(validator_index, public_key, ownership_proportion) VALUES(?, ?, ?)',
             [x.serialize_for_db() for x in validators],
         )
         self.db.update_last_write()
 
+    def edit_validator(self, validator_index: int, ownership_proportion: FVal) -> int:
+        cursor = self.db.conn.cursor()
+        cursor.execute(
+            'UPDATE eth2_validators SET ownership_proportion=? WHERE validator_index = ?',
+            (str(ownership_proportion), validator_index),
+        )
+        return cursor.rowcount
```

Comment:
This should always be either 1 or 0. Due to the primary key of the table it can't be more than 1. So here just simply raise `InputError()` with an error of editing non existing validator directly from this function.

Also add it and explain it the docstring.